diagrams/state_diagram.puml

```diff
@@ -0,0 +1,12 @@
+@startuml
+[*] --> Ordered: ClearableTransferOrdered
+Ordered --> InProcess: ClearableTransferInProcess
+Ordered --> Executed: ClearableTransferExecuted
+Ordered: tokens of payer are held and not available for transfer
+InProcess: tokens of payer are held and not available for transfer and the payer can not cancell the transfer
+Executed: the full or a partial transfer has been done
```

Comment:
The clearable can only do a full transfer, so the part with partial has to be removed

Reply:
You're right!